sources/startup.c

```diff
@@ -43,6 +43,7 @@
 #else
 #include <signal.h>
 #endif
+#include <execinfo.h>
```

Comment:
Because `execinfo.h` is not  part of the C standard library, this should be included only if `ENABLE_BACKTRACE` is enabled.